TTHAnalysis/macros/verifyAllFTrees.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+MODULE="leptonJetReCleanerSusyEWK2L"
+BASEPATH="/nfs/fanae/user/vischia/TREES_80X_011216_Spring16MVA_skim_2lep_OR_1lep_2tau/"
```

Comment:
It's not for the default values. I can even remove them.

But the verifyFTree as it is requires a single dataset name, while this script lets you specify a whole folder, which is far quicker and often what you actually want to do. 

I think it is an useful addition.